py/webdriver/edge/options.py

```diff
@@ -0,0 +1,226 @@
+# Licensed to the Software Freedom Conservancy (SFC) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The SFC licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import base64
+import os
+import platform
+import warnings
+
+from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
+
+
+class Options(object):
+    KEY = "ms:edgeOptions"
+
+    def __init__(self, use_chromium=False):
```

Comment:
Please remove the use_chromium_constructor parameter and use a getter/setter instead. This matches the code that is checked in to Selenium 4 alpha and will make it easier for users to upgrade their tests in the future. See https://github.com/SeleniumHQ/selenium/pull/8096 for reference.